src/api/spotify/auth.ts

```diff
@@ -1,6 +1,7 @@
 import pkg from 'crypto-js';
 const { SHA256, enc } = pkg;
-import type { MyResponseType, LocalStorage } from './types';
+import type { LocalStorage } from './types';
+import fetch from 'cross-fetch';
```

Comment:
fetch seems like it is a web-included standard and this seems really sus

Reply:
It is needed to run tests because fetch is not native to the Node.js enviornent.